.travis.yml

```diff
@@ -23,33 +24,34 @@ install:
   - composer update --prefer-source
 
 before_script:
-  - sh -c "if [ '$DB' = 'pgsql' ]; then sudo service postgresql stop; sudo service postgresql start $POSTGRESQL_VERSION; fi"
+  - if [ '$DB' = 'pgsql' ]; then sudo service postgresql stop; sudo service postgresql start $POSTGRESQL_VERSION; fi
 
 script: ./vendor/bin/phpunit --configuration tests/travis/$DB.travis.xml
 
 matrix:
   allow_failures:
-      - php: hhvm
-        env: DB=pgsql POSTGRESQL_VERSION=9.1
-      - php: hhvm
-        env: DB=pgsql POSTGRESQL_VERSION=9.2
-      - php: hhvm
-        env: DB=pgsql POSTGRESQL_VERSION=9.3
-      - php: hhvm
-        env: DB=sqlite
-      - php: hhvm
-        env: DB=mysql
-      - php: hhvm-nightly
+    - php: hhvm
+      env: DB=pgsql POSTGRESQL_VERSION=9.1
+    - php: hhvm
+      env: DB=pgsql POSTGRESQL_VERSION=9.2
+    - php: hhvm
+      env: DB=pgsql POSTGRESQL_VERSION=9.3
+    - php: hhvm
+      env: DB=sqlite
+    - php: hhvm
+      env: DB=mysql
+    - php: hhvm-nightly
   exclude:
-      - php: hhvm
-        env: DB=pgsql POSTGRESQL_VERSION=9.1 # driver currently unsupported by HHVM
-      - php: hhvm
-        env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
-      - php: hhvm
-        env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
-      - php: hhvm-nightly
-        env: DB=pgsql POSTGRESQL_VERSION=9.1 # driver currently unsupported by HHVM
-      - php: hhvm-nightly
-        env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
-      - php: hhvm-nightly
-        env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
+    - php: 7.0
```

Comment:
you are excluding all PHP 7 builds here again, meaning your change does not add them